rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -97,4 +110,34 @@ impl Run {
     pub fn attempt_count() -> u32 {
         LIVESPLIT_STATE.lock().unwrap().timer.run().clone().attempt_count()
     }
+    pub fn save_splits(path: String) -> Result<(), SplitsSaveError> {
+        let state = LIVESPLIT_STATE.lock().unwrap();
+        let mut timer = state.timer.clone();
+        let filename = Path::new(&path).file_name().unwrap().to_str().unwrap().to_owned();
+        let file = match File::create(path) {
+            Ok(file) => file,
+            Err(_e) => return Err(SplitsSaveError::CreationError(format!("ERROR: Failed to create {}!", filename))),
+        };
+        let writer = BufWriter::new(file);
+        match livesplit_core::run::saver::livesplit::save_timer(&mut timer, IoWrite(writer)) {
```

Comment:
Could you please import `livesplit_core::run::saver::livesplit`? That way the line becomes a more readable `match livesplit::save_timer(...) {`.